prod/manifest.json

```diff
@@ -1,36 +1,30 @@
 {
   "manifest_version": 2,
-
   "name": "codesy.io",
   "description": "codesy is a pay-what-you-want market for the open source community to encourage coders to fix important bugs.",
-  "version": "1.0",
-
+  "version": "2.0",
+  "bid_domain" : "api.codesy.io",
```

Comment:
I did this to be able to automate the domain the extension should work with.  Without it there's only the 'permissions' list and that could mean handling 1-N domains.  I'm in favor of leaving it in until we can think of a more elegant means.